core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -53,6 +59,24 @@
     private Function<? super Throwable, ?> responseCauseSanitizer = DEFAULT_CAUSE_SANITIZER;
     private Function<? super HttpHeaders, ?> responseTrailersSanitizer = DEFAULT_HEADERS_SANITIZER;
 
+    /**
+     * Sets the {@link Logger} to use when logging.
+     * If unset, a default {@link Logger} will be used.
+     */
+    public T logger(Logger logger) {
+        this.logger = requireNonNull(logger, "logger");
+        return self();
+    }
+
+    /**
+     * Returns the {@link Logger} the user specified to use,
+     * or {@code null} if not set and a default logger should be used.
```

Comment:
nit: I think we can remove this line.

Reply:
Never mind this. 😉

Reply:
I think it's fine as it is because it helps a reader understand under what condition `null` is returned.